HPXMLtoOpenStudio/tests/test_hvac_sizing.rb

```diff
@@ -134,8 +136,14 @@ def test_hvac_configurations
               elsif hp.fraction_cool_load_served == 0
                 assert_in_delta(htg_cap, htg_load, 1.0)
               else
-                assert_in_delta(htg_cap, [htg_load, clg_load].max, 1.0)
-                assert_in_delta(clg_cap, [htg_load, clg_load].max, 1.0)
+                if max_htg_cfm.nil? && max_clg_cfm.nil?
+                  assert_in_delta(htg_cap, [htg_load, clg_load].max, 1.0)
+                  assert_in_delta(clg_cap, [htg_load, clg_load].max, 1.0)
+                else
+                  # Check HP capacity is less than max(htg_load, clg_load)
+                  assert_operator(htg_cap, :<, [htg_load, clg_load].max, 1.0)
+                  assert_operator(clg_cap, :<, [htg_load, clg_load].max, 1.0)
```

Comment:
For `base-hvac-air-to-air-heat-pump-1-speed-existing-ductwork-restriction.xml`, we've restricted the airflow. So we'd expect that capacity is less than max of heating/cooling load.